src/java.net.http/share/classes/jdk/internal/net/http/Http1Response.java

```diff
@@ -139,8 +146,8 @@ public synchronized boolean acquire() {
             }
         }
 
-        public synchronized void tryRelease() {
-            if (state == 0x01) {
+        public void tryRelease() {
+            if (STATE.compareAndSet(this, (byte)0x01, (byte)0x03)) {
```

Comment:
Nit - a space is missing between the cast and the value for the second and third parameters.

Reply:
OK - will do - though I usually don't put a space there.

Reply:
It's fine in the current form if this wasn't an oversight.